### PR TITLE
fix: export xxhashBase64Url from wasm

### PR DESCRIPTION
**Pull request by sapphi-red**
This PR contains:

- [x] bugfix
- [ ] feature
- [ ] refactor
- [ ] documentation
- [ ] other

Are tests included?

- [x] yes (_bugfixes and features will not be merged without tests_)
- [ ] no

Breaking Changes?

- [ ] yes (_breaking changes will not be merged unless absolutely necessary_)
- [x] no

List any relevant issue numbers:

### Description

Rollup wasn't working in stackblitz because `xxhashBase64Url` was not re-exported from wasm.
Reproduction: https://stackblitz.com/edit/rollup-repro-ykzjaj?file=package.json

This PR fixes that and add a lint script to detect these kinds of mistakes.

P.S. I noticed the stackblitz template for the bug report uses rollup v3.

### CODE REVIEW

File: native.wasm.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+const { parse, xxhashBase64Url } = require('./wasm-node/bindings_wasm.js');
+
+exports.parse = parse;
+exports.xxhashBase64Url = xxhashBase64Url;
```
Comment (TrickyPi) [inline native.wasm.js:4]:
```suggestion
const { parse, xxhash_base64_url } = require('./wasm-node/bindings_wasm.js');

exports.parse = parse;
exports.xxhashBase64Url = xxhash_base64_url;
```
Currently, `wasm-pack` doesn't convert `under_score_case` to `camelCase`.

Comment (sapphi-red) [reply native.wasm.js:4]: Good catch! I felt renaming on the wasm side will make the code more consistent. So I did it in that way.

Comment (TrickyPi) [reply native.wasm.js:4]: Nice work! I wasn't aware of this attribute `js_name` before.

**Review by lukastaegert — APPROVED** [review]
Thanks so much!

**Pull request merged**